client/gutenberg/extensions/map-block/add-point/style.scss

```diff
@@ -0,0 +1,50 @@
+.map__add_btn {
```

Comment:
Seems like some classes don't follow Calypso's coding guidelines: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#general-syntax-and-writing-rules

Particularly, this one should be `.map__add-btn` instead.

Also, I'd suggest using our automatically generated class that's coming from `this.props.className` - it allows us to namespace our styles a bit better.